.github/workflows/main.yml

```
@@ -88,3 +88,74 @@ jobs:
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+      # 2. Clair Server
+      clair:
+        image: arminc/clair-local-scan:latest
+        ports:
+          - 6060:6060
+        depends-on:
```

actionlint (1.7.7)

unexpected key "depends-on" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)